contracts/interfaces/modules/royalty/policies/IRoyaltyPolicyLAP.sol

```diff
@@ -1,30 +1,26 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity 0.8.23;
 
-import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
-
 import { IRoyaltyPolicy } from "../../../../interfaces/modules/royalty/policies/IRoyaltyPolicy.sol";
 
 /// @title RoyaltyPolicy interface
```

Comment:
any more comments about this interface?

Reply:
Not sure if asking the others. 
If referring to inline comments: in the interfaces we have been showing only " /// @title " section without inline comments in a line below for example (kept for consistency)